.gitignore

```diff
@@ -1,3 +1,9 @@
 *.txt
 .vscode/
 vectors.json
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflicts got left in here? Let me try to resolve and then can you double-check?